src/sentry/web/frontend/choose_project.py

```diff
@@ -0,0 +1,38 @@
+from __future__ import absolute_import
+
+from django import forms
+from django.contrib import messages
+from django.core.urlresolvers import reverse
+from django.shortcuts import redirect
+
+from sentry.models import Project, Team
+from sentry.web.frontend.base import OrganizationView
+from sentry.utils.http import absolute_uri
+
+
+class OrganizationProjectChooser(OrganizationView):
+    required_scope = 'team:read'
+
+    def handle(self, request, organization):
+        teams = Team.objects.get_for_user(
+            organization=organization,
+            user=request.user,
+            with_projects=True,
+            )
+
+        # next_url should have a trailing slash only: settings/install/
+        next_url = request.GET.get('next')
```

Comment:
It's also possible that this would return `None`, causing `next_url[0]` to raise a `TypeError`